packages/global-styles/src/components/font-pairing-variations/index.tsx

```diff
@@ -82,25 +83,33 @@ const FontPairingVariations = ( {
 		<Composite
 			{ ...composite }
 			role="listbox"
-			className="font-pairing-variations"
 			aria-label={ translate( 'Font pairing variations' ) }
 		>
-			<FontPairingVariation
-				key="base"
-				fontPairingVariation={ base }
-				isActive={ ! selectedFontPairingVariation }
-				composite={ composite }
-				onSelect={ () => onSelect( null ) }
-			/>
-			{ fontPairingVariations.map( ( fontPairingVariation, index ) => (
+			<h3 className="global-styles-variation__title">{ translate( 'Free fonts' ) }</h3>
```

Comment:
Same here. It might be better to display `Free font`

Reply:
I don't have a strong opinion. I just think we have to keep consistency and we have an [issue](https://github.com/Automattic/wp-calypso/issues/78105) related to `Premium styles` before as well. Your call 😆

Reply:
Thanks for the context 😂 I'll fix it than